core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -501,6 +501,13 @@ public String getMetadataPath() {
     return CarbonTablePath.getMetadataPath(getTablePath());
   }
 
+  /**
+   * Return the stage input path
+   */
+  public String getStagePath() {
```

Comment:
CarbonTablePath.`SNAPSHOT_FILE_NAME` is not used. can we remove it ?

Reply:
It is used in func `getStageSnapshotFile`

Reply:
I know, that function is used only for test files. I didn't see `snapshot` file creation in the flink sdk write code.